src/client/components/chart-line/chart-line.tsx

```diff
@@ -32,109 +32,107 @@ export interface ChartLineProps {
   scaleX: (v: any) => number;
   scaleY: (v: any) => number;
   color: string;
+  dashed?: boolean;
```

Comment:
I would use enum instead of boolean, I could imagine more line styles than default and dashed.

Reply:
Two things here: Different styles won't map to directly to one value, yet to some complex map of CSS props. Also different styles could be applicable together.